srcpkgs/rogue/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'rogue'
+pkgname=rogue
+version=5.4.4
+revision=1
+archs="x86_64-glibc"
+makedepends=ncurses-devel
+depends="ncurses"
+short_desc="Dungeon crawl game from the 80s"
+maintainer="sirkhancision <jsantiago12tone@gmail.com>"
+license="custom:Rogue"
```

Comment:
```suggestion
license="BSD-3-Clause"
```

Reply:
Thanks for this one, had some trouble finding the license because I only really looked inside the source files.